selendroid-server/src/main/java/io/selendroid/server/android/internal/Rectangle.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2012-2014 eBay Software Foundation and selendroid committers.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.selendroid.server.android.internal;
+
+/**
+ * A copy of java.awt.Rectangle, to remove dependency on awt.
+ */
+public class Rectangle {
```

Comment:
I don't think we really need this class Android already has a Rect implementation
https://developer.android.com/reference/android/graphics/Rect.html